sympy/crypto/tests/test_crypto.py

```diff
@@ -16,7 +16,7 @@
       encipher_gm, gm_public_key, gm_private_key, encipher_bg, decipher_bg,
       bg_private_key, bg_public_key, encipher_rot13, decipher_rot13,
       encipher_atbash, decipher_atbash, NonInvertibleCipherWarning,
-      encipher_railfence, decipher_railfence)
+      encipher_railfence, decipher_railfence, decipher_columnar)
```

Comment:
Can you add a test for `encipher_columnar`?

Reply:
Test Added :) 
I did not add the test earlier because encryption was adding random letters to the cipher but I I just realised that the workaround was to make the length of the message divisible by length of key :)